mkimage/debootstrap

```diff
@@ -203,7 +203,10 @@ fi
 	# delete all the apt list files since they're big and get stale quickly
 	rm -rf "$rootfsDir/var/lib/apt/lists"/*
 	# this forces "apt-get update" in dependent images, which is also good
-	echo "deb http://archive.raspberrypi.org/debian $suite main ui" >>  "$rootfsDir/etc/apt/sources.list.d/raspi.list"
-	chroot "$rootfsDir" bash -c 'sudo apt-key adv --batch --keyserver ha.pool.sks-keyservers.net  --recv-key 0x82B129927FA3303E'
+	if [ $suite != 'bullseye' ]; then
```

Comment:
https://github.com/balena-io-library/base-images/issues/602 suggests that a similar check is needed for `sid` too? :man_shrugging: